templates/helm/templates/leader-election-role-binding.yaml.tpl

```diff
@@ -14,6 +14,6 @@ roleRef:
   name: {{.ServicePackageName}}-leader-election-role
 subjects:
 - kind: ServiceAccount
-  name: {{.ServiceAccountName}}
+  name: {{ include "service-account.name" . }}
```

Comment:
@grandwizard28 since this is template generating helm templates we'll have to do something like
```
{{ "{{ include "service-account.name" . }}" }}
```

Reply:
Oops, my bad!